api/openapi/solidus-api.oas.yml

```diff
@@ -6174,18 +6174,26 @@ components:
           type: string
         order_number:
           type: string
+    promotion-handler-error:
+      type: object
+      title: Promotion handler error
+      properties:
+        error:
+          type: string
+        error_code:
+          type: string
     coupon-code-handler:
       type: object
       title: Coupon code handler
       properties:
         success:
           type: string
-        error:
```

Comment:
We should keep this and mark it as deprecated, it will be removed in the next major bump.